runtime/oti/jclprots.h

```diff
@@ -800,10 +802,7 @@ jcharArray JNICALL Java_com_ibm_oti_io_NativeCharacterConverter_convertBytesToCh
 
 /* BBjclNativesWin32SystemHelpers*/
 char* getPlatformFileEncoding (JNIEnv *env, char *codepage, int size, int encodingType);
-I_32
-convertToUTF8 (J9PortLibrary* portLibrary, const wchar_t* unicodeString, char* utf8Buffer, UDATA size);
 char * getTmpDir (JNIEnv *env, char **tempdir);
```

Comment:
Please remove the declarations of `getTmpDir()` from this file.
The file `attach.c` should include `util_api.h` instead of declaring `getTmpDir()`.

Reply:
Updated for the last three comments.